### DISCUSSION TITLE
infinispan-secret failing during install

### DISCUSSION THREAD

**Discussion by skumaravelan** [apache/camel-karavan#923] [General]
Hi Marat, 
I am getting the below error when I am starting my installation. Could you please check and advise.

19m         Warning   Failed             pod/karavan-65d8d4697d-jqrrx    Error: secret "infinispan-secret" not found
19m         Normal    Pulled             pod/karavan-65d8d4697d-jqrrx    Successfully pulled image "docker.apple.com/apache/camel-karavan:4.0.1" in 63.216358ms (63.226206ms including waiting)

Thanks
Senthil

**Comment by skumaravelan** [comment]
I have found a solution for this one.  Now  I have infinispan working .

**Reply by oscerd** [reply]
Sharing your solution could help others. It's how an open source project works

**Comment by skumaravelan** [comment]
`--infinispan-image` - Infinispan Image
`--infinispan-username` - Infinispan Username
`--infinispan-password` - Infinispan Password
`--install-infinispan` - Install Infinispan

**Comment by pai911** [comment]
I'm seeing the same issue.

and I can't seem to fix the issue even after adding these params...

```
--install-infinispan 
--infinispan-image quay.io/infinispan/server:14.0.21.Final 
--infinispan-username user 
--infinispan-password pass
```

I still see `Warning  Failed     5s (x3 over 17s)  kubelet            Error: secret "infinispan-secret" not found`.

What exactly is this `infinispan` used for in Karavan? Is it really required?
